tests/features/collection/collection_overview.feature

```diff
@@ -2,17 +2,13 @@
 Feature: Collections Overview
 
   Scenario: Check visibility of "Collections" menu link.
-    Given I am an anonymous user
-    Then I should see the link "Collections"
-    When I click "Collections"
-    Then I should see the heading "Collections"
-    # Check that all logged in users can see and access the link as well.
+    # Check that all logged in users can see and access the link.
```

Comment:
Nevermind I saw the designs. Still, you should not simply remove it. It is removed because we are changing the homepage. A ticket about implementation of the homepage should be referred here to justify the changes.